package.json

```diff
@@ -7,7 +7,7 @@
   "repository": "git@github.com:awslabs/iot-app-kit.git",
   "engines": {
     "node": ">=16.0.0",
-    "npm": ">=8.0.0"
+    "npm": ">=8.5.0"
```

Comment:
https://github.com/npm/cli/issues/3637

There is a bug in NPM that was fixed in 8.5.0 which prevents `npm install` from erroring out when finding packages that don't exist in the registry but DO exist in the workspace